scripts/lib/create-zulip-admin

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if ([ "$ZULIP_USER_CREATION_ENABLED" == "True" ] || [ "$ZULIP_USER_CREATION_ENABLED" == "true" ]) && ([ -z "$ZULIP_USER_DOMAIN" ] || [ -z "$ZULIP_USER_EMAIL" ] || [ -z "$ZULIP_USER_FULLNAME" ] || [ -z "$ZULIP_USER_PASS" ]); then
```

Comment:
Can we wrap this line?  It's giant absurdly long :)

Reply:
Wrapped.